collections/duties.js

```diff
@@ -86,6 +86,46 @@ function deleteDutie(id) {
   });
 }
 
+function updateDutie(data, id) {
+  const resHandler = require("../start").server.getResHandler();
+  const collection = require("../start").dutiesCollection;
+
+  if (!utils.isMatchesRegex(/[0-9a-fA-F]{24}/, id)) {
+    resHandler.routeNotFound("the id that was sent is not legal, it needs to be of 24 chars in alpha-numeric");
```

Comment:
this sure is a common error message for this method. how about a semi method called "illegal id" that simply calls this method with this error id